libs/metrics-wai/src/Data/Metrics/Middleware/Prometheus.hs

```diff
@@ -33,12 +34,17 @@ import Network.Wai.Routing.Route (Routes, prepare)
 -- This middleware requires your servers 'Routes' because it does some normalization
 -- (e.g. removing params from calls)
 waiPrometheusMiddleware :: (Monad m) => Routes a m b -> Wai.Middleware
-waiPrometheusMiddleware routes =
+waiPrometheusMiddleware routes = waiPrometheusMiddlewarePaths $ treeToPaths $ prepare routes
+
+-- | Helper function that should only be needed as long as we have wai-routing code left in
```

Comment:
I'm wondering if such long Haddock titles are rendered nicely? :thinking:

Reply:
Argh, my fault, sorry!

Haddock has no notion of a title. (I wrongly thought these would be in the first line.)